atlassian_jwt_auth/contrib/flask_app/decorators.py

```diff
@@ -17,26 +16,17 @@ def requires_asap(f):
     @wraps(f)
     def decorated(*args, **kwargs):
         verifier = _get_verifier()
```

Comment:
Nit: could `_get_verifier` be passed to `_requires_asap` too?

Reply:
It could but I didn't see a need to do so.

Reply:
Typo on my part: `s/too/instead of verifier/`

Since both decorators already abstract getting the verifier into a function call, we may as well just pass the function to `_requires_asap` directly.

Reply:
I see no need. Why shouldn't callers be able to provide the verifier?